gitpod-io/gitpod
# Organization timeout defaults

## Description

Makes default workspace timeouts configurable on the organization level. Cool, eh?

## Related Issue(s)

Fixes ENT-604

## How to test


Try to change them and start a workspace. Note that you'll need to upgrade to a PAYG plan to access the feature.

https://ft-org-timb9230e944a.preview.gitpod-dev.com/settings

/hold
